api-reference/receipts/get-started.markdown

```diff
@@ -7,21 +7,32 @@ layout: reference
 
 ## Get Started
 
+- [Overview](#overview)
 - [Version](#version)
 - [Regional Availability](#regional-availability)
 - [Explore the API](#explore-the-api)
   - [Prerequisites](#prerequisites)
-  - [Endpoints](#endpoints)
-    - [Acquire an Access Token](#retrieve-a-user-access-token)
-    - [GET Service Index](#endpoint-service-index)
-    - [GET Schemas](#endpoint-schemas)
-    - [POST Receipts](#endpoint-post-a-receipt)
-    - [GET Receipts by User ID](#endpoint-get-receipts-by-userid)
-    - [GET Receipts by Receipt ID](#endpoint-get-a-receipt-by-id)
-    - [GET Receipt Image by Receipt ID](#endpoint-get-receipt-image)
-    - [GET Receipt Status by Receipt ID](#endpoint-get-receipt-status)
+  - [Acquire an Access Token](#retrieve-a-user-access-token)
   - [Explore the API in JavaScript](#explore-the-api-in-javascript)
 
+### Overview
+
+The Receipts V4 API aims to make our service as useful as possible. In order to do so we accept three different formulae for posting a receipt:
+
+1. __Receipt Data__ - Your receipt data is stored along with an automatically generated receipt image file.
+2. __Receipt Data & Receipt Image__ - Your receipt data and receipt image file are stored.
+3. __Receipt Image w/o Data__ - Your receipt image file is stored along with some accompanying metadata.
+
+All of the above are receipt resources, but the service draws a distinction between resources with data versus resources that are standalone images.
+
+Resources with data are schema-enforced and are referred to as eReceipts.
+
+Resources of standalone images are referred to as as Image-Only Receipts.
+
+These two different resources are sent/fetched from the Receipts V4 API via different endpoints:
+- eReceipts (Receipts With Data) - Use eReceipt Endpoints
+- Image-Only Receipts (Standalone Images Without Data) - Use Image-Only Receipt Endpoints
+
 ### Version
 
 - 4.0
```

Comment:
In reference to Regional Availability, are we ready to add the external URL for China?

Reply:
We discussed this and decided to do it separately :)